SalesforceFoundation/HEDAP
# Feature/xinnanli/account name format

# Critical Changes

# Changes
We now support customizing the format of administrative account and household account names. In HEDA Settings, on the System tab, choose a preset naming option or create your own custom naming convention. The name of a household account will be automatically updated when a new contact is added to the household account or an existing contact is renamed.
# Issues Closed
Fixes #218 
# New Metadata
CMP_Acc_Naming_Dropdown
Hierarchy_Settings__c.Admin_Account_Naming_Format__c
Hierarchy_Settings__c.Admin_Other_Name_Setting__c
Hierarchy_Settings__c.Household_Account_Naming_Format__c
Hierarchy_Settings__c.Household_Other_Name_Setting__c
None
acctNamingOther
adminAccNameFormat
firstNameLastNameAdminACC
firstNameLastNameFamily
firstNameLastNameHH
hhAccNameFormat
hhFamily
lastNameAdminAcc
 lastNameFamily
 lastNameFirstNameAdminAcc
 lastNameFirstNameFamily
lastNameFirstNameHH
lastNameHH
# Deleted Metadata

# Testing Notes
1. Deploy the HEAD package to Dev org or Scratch org
2. Go to HEDA setting page and switch to system tab
3. Select Admin account or Household account as default record type
4. Select naming setting for admin account and household account
4. Create a contact and make sure the name of account is correct
